README.md

```diff
@@ -169,7 +169,8 @@ docker compose run app manage compilemessages
 
 ### Code style
 
-Our python code is linted using [`black`](https://github.com/psf/black), [`isort`](https://github.com/PyCQA/isort) and [`autoflake`](https://github.com/myint/autoflake).
+Our python code is linted using [`ruff`](https://docs.astral.sh/ruff/).
```

Comment:
I'd go further and explain that Ruff also takes care of formatting the code and sorting imports